osquery/tables/system/windows/wmi_bios_info.cpp

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2014-present, The osquery authors
+ * Copyright (c) 2021-present, The osquery authors
```

Comment:
Please don't change this line

Reply:
Hi!
Oops, sorry about the year change. I'll fix that, didn't notice.

Yeah, initially I thought we should support all Dell machines, but again I wasn't sure what the scope for osquery's support was going to be here (especially for machines released before 2018).